lifecycle-operator/controllers/lifecycle/keptnworkloadversion/controller_test.go

```diff
@@ -1142,16 +1144,23 @@ func setupReconciler(objs ...client.Object) (*KeptnWorkloadVersionReconciler, ch
 	}}
 
 	fakeClient := fake.NewClient(objs...)
-
 	recorder := record.NewFakeRecorder(100)
+
 	r := &KeptnWorkloadVersionReconciler{
 		Client:        fakeClient,
 		Scheme:        scheme.Scheme,
 		EventSender:   controllercommon.NewK8sSender(recorder),
 		Log:           ctrl.Log.WithName("test-appController"),
-		TracerFactory: tf,
 		Meters:        controllercommon.InitAppMeters(),
 		SpanHandler:   &telemetry.SpanHandler{},
+		TracerFactory: tf,
+		EvaluationHandler: &evalfake.MockEvaluationHandler{
+			ReconcileEvaluationsFunc: func(ctx context.Context, phaseCtx context.Context, reconcileObject client.Object, evaluationCreateAttributes evaluation.CreateEvaluationAttributes) ([]klcv1alpha3.ItemStatus, apicommon.StatusSummary, error) {
+				var itemStatus []klcv1alpha3.ItemStatus
+				var summary apicommon.StatusSummary
+				return itemStatus, summary, nil
```

Comment:
Here we can directly return empty array or empty summary struct, we do not need to create variables and return them afterwards

Reply:
I have made the fix here.